benchmarks/fixtures/query.js

```diff
@@ -23,17 +24,21 @@ function setupKeystone(adapterName) {
 
 const group = new FixtureGroup(setupKeystone);
 
+const createUser = keystone =>
+  createItem({
+    keystone,
+    item: { name: 'test', posts: { create: [] } },
+  });
+
+const createUsers = (keystone, users) =>
+  createItems({
+    keystone,
+    item: users,
+  });
+
 group.add({
   fn: async ({ keystone, adapterName }) => {
-    const { data } = await graphqlRequest({
-      keystone,
-      query: `
-    mutation {
-      createUser(data: { name: "test", posts: { create: [] } }) { id }
-    }`,
-    });
-    const userId = data.createUser.id;
-
+    const { id: userId } = await createUser(keystone);
```

Comment:
I think it would be easier to follow this code if the calls to `createItem` and `createItems` were inline, rather than having them abstracted away behind yet another function.